_posts/2016-08-01-tips-for-freshmen-by-nuswhispers.markdown

```diff
@@ -0,0 +1,144 @@
+The number of good friends I've made in NUS can be counted on one hand. But they have provided me with valuable information, laughter, HTHT sessions and whatnot. These friendships were all forged in the above mentioned places and sadly not in camps ha ha. In life, you only need a few good friends that have your back when you need it. That and your Significant Other HA HA.  
+1. I believe I triggered the OCD in some of you when you noticed bullet point one was missing at the beginning of this post.
+That is good. Attention to detail is very important. Ok i digress again.
+>
+>Anyway you can skip the previous points mentioned because this point is the most important.
+And that is to enjoy what you are learning in school. If you don't like your major, change it don't drag. If you find some modules you've taken to be boring, make it interesting (or drop it before week 5?). Don't dwell on the past (failures? bad grades?) or worry too much about the future. Enjoy learning in the present. It is a gift. Time lost which cannot be recovered.
+Anyway this post is freaking long so imma just peace out.. gotta live my life now :D
+>
+> Cheers,
+>
+> Your senior
+
+
+- Get the Proxy Bookmarklet installed on your laptops/desktops. The instructions for installation on both Windows/Mac devices can be found here: <http://libguides.nus.edu.sg/proxybookmarklet/installondesktop>. Once installed, if you ever come across a journal article in an academic journal that you wish to access, just go to your bookmarks and click on the proxy bookmarklet to gain free access to the article by signing in with your NUSNET ID. These articles help to complement your learning, and also help one with writing good essays/assignments. [[NUSWhispers #22833](https://www.nuswhispers.com/confession/22833)]
+
+- Get EndNote! This is a referencing software that helps you to manage and format your citations when you are writing your assignments and essays. They have a ton of reference styles uploaded within their database and you’ll never need to type out your references manually ever again. Without EndNote, I would have never finished my thesis. Instructions for installations are found here: <http://libguides.nus.edu.sg/endnote/installendnote >[[NUSWhispers #22833](https://www.nuswhispers.com/confession/22833)]
+
+- If you don't wish to splash out money on a new textbook, search the used textbook forum in IVLE or just use the RBR copy. [Han Ming Guang]
+
+- Here is how you get access of textbooks legally without spending extra money:  
+1. Go to NUS library website (google "NUS library")
+2. Type in the name of the book and Authors on the search bar
+3. If you found the book and it the word "available online", then congratulations, free book! If not, then too bad. But it should work for most textbooks though.  
+[Goh Koon Tong]
+
+<!-- -->
```

Comment:
OH LOL okay cool, in that case can we add something like "Force a new line between the point above and the one below" so we know :) Thanks!